### PR TITLE
Issue 1411 rights

### PR DESCRIPTION
**Pull request by Maxstgt**
part one of the issue ref: #1411

### CODE REVIEW

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/technical-info/technical-info.component.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { Component, ChangeDetectionStrategy } from '@angular/core';
+
+@Component({
+    selector: 'catalog-tunnel-technical-info',
+    templateUrl: './technical-info.component.html',
+    styleUrls: ['./technical-info.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TunnelTechnicalInfoComponent {
+    constructor(public movieForm: MovieForm) { }
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/dashboard/movie-tunnel/technical-info/technical-info.component.ts]: For consistency, call it `form`

File: libs/movie/src/lib/movie/form/version-info/version-info.component.html [hunk 2]
```diff
@@ -1,28 +1,33 @@
...
-      </mat-option>
-      <mat-option *ngFor="let language of languages$ | async" [value]="language.slug">
-        {{ language.label }}
-      </mat-option>
+      </mat-option> 
     </mat-select>
   </mat-form-field>
+  
+  <!-- Specify the version -->
+  <div class="mat-body-2">Version</div>
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/form/version-info/version-info.component.html]: use span or p.

File: libs/movie/src/lib/movie/form/version-info/version-info.component.html [hunk 3]
```diff
@@ -1,28 +1,33 @@
...
+  <div class="caption"> Select all that apply</div>
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/form/version-info/version-info.component.html]: If this is a caption why not using the tag `<caption>` or another inline text tag like span.

File: libs/movie/src/lib/movie/form/version-info/version-info.component.html [hunk 4]
```diff
@@ -1,28 +1,33 @@
...
+  <mat-button-toggle-group name="language" aria-label="Language" multiple color="primary">
+    <mat-button-toggle value="dubbed">Dubbed</mat-button-toggle>
+    <mat-button-toggle value="subtitled">Subtitled</mat-button-toggle>
+    <mat-button-toggle value="closed-captioned">Closed-captioned</mat-button-toggle>
+  </mat-button-toggle-group>
+
+  <!-- Add new language-->
+  <div fxLayout="column" fxLayoutAlign="center center" class="full-width add-btn">
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/form/version-info/version-info.component.html]: Whould be nice to reformat those lines.

File: libs/movie/src/lib/movie/form/version-info/version-info.component.ts [hunk 5]
```diff
@@ -1,38 +1,38 @@
...
       debounceTime(200),
       distinctUntilChanged(),
-      map(name => this.staticModels['LANGUAGES'].filter(item => item.label.toLowerCase().indexOf(name.toLowerCase()) > -1))
+      map(name => this.staticModels.filter(item => item.label.toLowerCase().indexOf(name.toLowerCase()) > -1))
     );
   }
 
-  get versionInfo(): MovieVersionInfoForm {
-    return this.controlContainer.control as MovieVersionInfoForm;
+  get addLanguageVersion() {
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/form/version-info/version-info.component.ts]: Remove or add comment with todo referencing your issue.

File: libs/movie/src/lib/movie/form/version-info/version-info.component.html (hunk 4 above)
Comment (Maxstgt) [reply libs/movie/src/lib/movie/form/version-info/version-info.component.html]: it is formatted with the rules of prettier

**Pull request merged**